ddtrace/internal/telemetry/logging.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import os
+import traceback
+
+from ddtrace.internal.telemetry.constants import TELEMETRY_LOG_LEVEL
+
+
+class DDTelemetryLogHandler(logging.Handler):
```

Comment:
Do we care if we miss errors because of the rate limiting logic in `ddtrace.internal.logger.log_filter` ?

it would mean counts are incorrect, but should mean we get the `{}, %s additional messages skipped` suffix on the log record though (actually would this be a problem?)